freqtrade/data/btanalysis.py

```diff
@@ -105,6 +108,23 @@ def get_latest_hyperopt_file(directory: Union[Path, str], predef_filename: str =
     return directory / get_latest_hyperopt_filename(directory)
 
 
+def load_backtest_metadata(filename: Union[Path, str]) -> Dict[str, Any]:
+    """
+    Read metadata dictionary from backtest results file without reading and deserializing entire
+    file.
+    :param filename: path to backtest results file.
+    :return: metadata dict or None if metadata is not present.
+    """
+    filename = get_backtest_metadata_filename(filename)
+    try:
+        with filename.open() as fp:
+            return json_load(fp)
+    except FileNotFoundError:
+        return {}
```

Comment:
The Premise of this looks good - but it doesn't seem to work.

I've just ran 3 backtests in a row (all produce the same hash based on the .meta file) - but it doesns't work and always runs into this.
Some debugging shows that the "loading" filename is generated wrongly.
I think for loading, you'll not need to call `get_backtest_metadata_filename` as the ".glob" already returns the correct filename.